imagevector/images.yaml

```
@@ -129,7 +129,7 @@ images:
   # NOTE: Please make sure to copy new image versions when updating the version by adding them to
   #       https://github.com/gardener/ci-infra/blob/master/config/images/images.yaml.
   repository: europe-docker.pkg.dev/gardener-project/releases/3rd/eks/aws-load-balancer-controller
-  tag: "v2.7.1"
+  tag: "v2.8.1"
```

```suggestion
  tag: "v2.8.2"
```

There is already a new version, but not yet replicated into our repository.
see  https://github.com/gardener/ci-infra/pull/2188